graph/cytoscape/cytoscape.go

```diff
@@ -32,17 +32,20 @@ type NodeData struct {
 	// App Fields (not required by Cytoscape)
 	Service           string `json:"service"`
 	Version           string `json:"version,omitempty"`
-	Rate              string `json:"rate,omitempty"`           // edge aggregate
-	Rate3xx           string `json:"rate3XX,omitempty"`        // edge aggregate
-	Rate4xx           string `json:"rate4XX,omitempty"`        // edge aggregate
-	Rate5xx           string `json:"rate5XX,omitempty"`        // edge aggregate
-	RateSelfInvoke    string `json:"rateSelfInvoke,omitempty"` // rate of selfInvoke
-	HasCircuitBreaker string `json:"hasCB,omitempty"`          // true | false
-	HasRouteRule      string `json:"hasRR,omitempty"`          // true | false
-	IsDead            string `json:"isDead,omitempty"`         // true (has no pods) | false
-	IsGroup           string `json:"isGroup,omitempty"`        // set to the grouping type, current values: [ 'version' ]
-	IsRoot            string `json:"isRoot,omitempty"`         // true | false
-	IsUnused          string `json:"isUnused,omitempty"`       // true | false
+	Rate              string `json:"rate,omitempty"`              // edge aggregate
+	Rate3xx           string `json:"rate3XX,omitempty"`           // edge aggregate
+	Rate4xx           string `json:"rate4XX,omitempty"`           // edge aggregate
+	Rate5xx           string `json:"rate5XX,omitempty"`           // edge aggregate
+	RateSelfInvoke    string `json:"rateSelfInvoke,omitempty"`    // rate of selfInvoke
+	HasCircuitBreaker string `json:"hasCB,omitempty"`             // true | false
+	HasRouteRule      string `json:"hasRR,omitempty"`             // true | false
+	IsDead            string `json:"isDead,omitempty"`            // true (has no pods) | false
+	IsGroup           string `json:"isGroup,omitempty"`           // set to the grouping type, current values: [ 'version' ]
+	IsRoot            string `json:"isRoot,omitempty"`            // true | false
+	IsUnused          string `json:"isUnused,omitempty"`          // true | false
+	PodsCount         int    `json:"podsCount,omitempty"`         // integer
```

Comment:
Do we really need 3 new fields for the results of the sidecar check?  I'm not sure of the client-side requirements for this PR but unless we have to have it I'd prefer that we stay with our general approach of using the graph to flag areas of interest, and let the user drill down into the details page.  Maybe a single field like HasMissingSidecars with either a boolean value, or possibly a string with the number of missing sidecars (i.e. PodsCount-PodsWithSidecars)

Reply:
I can use the single field HasMissingSidecars. I'll go that way.
Probably, the other data won't be too useful in the graph. This will simplify a little the code.